tests/phpunit/tests/rest-api/rest-themes-controller.php

```diff
@@ -555,6 +575,17 @@ public function test_theme_template() {
 		$this->assertSame( 'default', $result[0]['template'] );
 	}
 
+	/**
+	 * @ticket 61021
+	 */
+	public function test_theme_template_uri() {
+		$response      = self::perform_active_theme_request();
```

Comment:
Would it worth testing inactive themes as well after b1dd7ff35e3da3adc786ded589bd99d66bcd643e?

Reply:
Oh, thanks for spotting that! I've updated the tests in https://github.com/WordPress/wordpress-develop/pull/6399/commits/b44448932999ae718d570acbeccedcbdaf71638e to include both active and inactive themes in the response, and loop over the results to check depending on the status.

Let me know if you think it's better to split into separate tests, though.